janus_server/src/task.rs

```diff
@@ -136,6 +138,7 @@ pub struct Task {
     pub(crate) collector_hpke_config: HpkeConfig,
     /// Key used to authenticate messages sent to or received from the other
     /// aggregators.
+    #[derivative(Debug = "ignore")]
     pub(crate) agg_auth_keys: Vec<AggregatorAuthKey>,
     /// HPKE configurations & private keys used by this aggregator to decrypt client reports.
     pub(crate) hpke_keys: HashMap<HpkeConfigId, (HpkeConfig, HpkePrivateKey)>,
```

Comment:
We should omit the `HpkePrivateKey` values from the debug output, too, but I'm not sure how to do that without replacing this `HashMap` with some custom type.

Reply:
This is using our `HpkePrivateKey` type, which currently uses a derived `Debug` implementation. We could rewrite its implementation to be opaque, then we would get the behavior we want here. (The same tactic would work on `AggregatorAuthKey`) If we ever wanted to dump a key in a unit test, we'd have to take another step or two to get the key's inner bytes.

Reply:
I wrote up #144 for this.